core/src/main/java/com/volmit/iris/core/service/StudioSVC.java

```diff
@@ -369,6 +369,7 @@ public void close() {
     }
 
     public File compilePackage(VolmitSender sender, String d, boolean obfuscate, boolean minify) {
+        System.out.println(new File(getWorkspaceFolder(), d) + " -> " + new File(getWorkspaceFolder(), d).exists());
```

Comment:
don't use `System.out`, use Iris::info instead

Reply:
Whoa, my fault. This line is written for debugging, I forgot to remove it